po/es.po

```diff
@@ -292,8 +292,8 @@ msgstr "Subtítulos"
 
 #: ui/preferences-window.ui:144
 msgid "Default font"
-msgstr "Fuente predeterminada"
-
+msgstr "Fuente por defecto"
+ 
```

Comment:
nitpick: Invisible space was added.

Reply:
Please remove this added space at the line beginning. Its probably harmless, but better not add it to be safe.